src/chocolatey/GetChocolatey.cs

```diff
@@ -49,8 +49,9 @@ public class GetChocolatey
 
         public GetChocolatey()
         {
+            Log4NetAppenderConfiguration.configure();

```

Comment:
Would it be better to call `Bootstrap.Initialize` instead of directly initializing the logger?